Sources/KituraWebSocket/WSFrameParser.swift

```diff
@@ -81,8 +81,15 @@ struct WSFrameParser {
     
     private mutating func parseOpCode(bytes: UnsafePointer<UInt8>, from: Int) -> (WebSocketError?, Int) {
         let byte = bytes[from]
-        frame.finalFrame = byte & 0x80 != 0
+        // 0x.. is the hexidecimal representation of the bits you would like to check
+        // e.g.0x0f is 15 which is 00001111 so rawOpCode is the last 4 digits
         
+        // Check RSV is equal to 0
+        let rsv = (byte & 0x70) >> 4
```

Comment:
Why do we need to shift?

Reply:
so from the first byte, rsv bits are at 01110000. The shift makes the int[8] value of rsv represent 0-7 instead of 0, 16, 32, 48, 64. In our case since we are just ensuring they are 0 this doesn't matter but seemed sensible if we are RSV in future.

Reply:
I see. I'm not sure shifting is meaningful really - if we ever need to support RSV bits in future we can just mask them as needed. We're only ever going to refer to them as part of a UInt8 byte including the opcode, aren't we?